bundles/mapping/mapmodule/MapModuleClass.ol.js

```diff
@@ -1186,6 +1186,11 @@ export class MapModule extends AbstractMapModule {
         return olGeoJSON.writeFeaturesObject(features);
     }
 
+    setTime () {
+        var log = Oskari.log('Oskari.mapframework.ui.module.common.MapModule');
+        log.warn('setTime only available in 3D map');
```

Comment:
This is where it's being handled for 2d maps. Should this be put to requestHandler instead where it checks that 3dmap is enabled?

Reply:
This is ok.

Reply:
On the otherhand if notifyTimeChanged() is in AbstractMapModule.js I think this should be in there too.

Reply:
I could move the notifyTimeChanged to be olcs specific